advanced/pihole-FTL.systemd

```diff
@@ -0,0 +1,40 @@
+[Unit]
+Description=Pi-hole FTLDNS
+; This unit is supposed to indicate when network functionality is available, but it is only
+; very weakly defined what that is supposed to mean, with one exception: at shutdown, a unit
+; that is ordered after network.target will be stopped before the network
+After=network.target
+Requires=network.target
```

Comment:
The Requires line should be dropped.
network.target is a so-called passive target (see man systemd.special) and should not be pulled in actively by units.
It is pulled in by the service implementing network.target. Other units should simply order itself against network.target.